apollo-test/src/main/java/com/spotify/apollo/test/ServerHelperSetup.java

```diff
@@ -0,0 +1,45 @@
+/*
+ * -\-\-
+ * Spotify Apollo Testing Helpers
+ * --
+ * Copyright (C) 2013 - 2015 Spotify AB
```

Comment:
Nitpick, guess the copyright end year should be updated?

Reply:
Oh, right I haven't noticed this